docs/installation/rhel.md

```diff
@@ -12,150 +12,178 @@ parent = "smn_linux"
 
 Docker is supported on the following versions of RHEL:
 
-- [*Red Hat Enterprise Linux 7 (64-bit)*](#red-hat-enterprise-linux-7-installation)
-- [*Red Hat Enterprise Linux 6.6 (64-bit)*](#red-hat-enterprise-linux-66-installation) or later
+- Red Hat Enterprise Linux 7 
+- Red Hat Enterprise Linux 6.6 or later
```

Comment:
I'm not the person to ask.  I took what was on the page in 1.6 as a starting point: http://docs.docker.com/installation/rhel/